src/config/ch.sbb.netzkarte/index.js

```diff
@@ -184,111 +184,94 @@ export const punctuality = new Layer({
-    visible: false,
-    tenant: 'sbb',
-    getMotsByZoom: (zoom) => (zoom < 11 ? [] : ['tram', 'subway']),
-    properties: {
-      isQueryable: true,
-      popupComponent: 'PunctualityPopup',
-      useTrackerMenu: true,
-    },
-  }),
-  new TralisLayer({
-    isUpdateBboxOnMoveEnd: true,
-    name: 'ch.sbb.puenktlichkeit-nv',
-    visible: false,
-    tenant: 'sbb',
-    getMotsByZoom: (zoom) => (zoom < 4 ? [] : ['rail']),
-    filter: ({ properties }) => {
-      const { type, line } = properties;
-      return (
-        type &&
-        line?.name &&
-        type === 'rail' &&
-        /^(S|R$|RE|PE|D|IRE|RB|TER)/.test(line.name)
-      );
-    },
-    properties: {
-      isQueryable: true,
-      popupComponent: 'PunctualityPopup',
-      useTrackerMenu: true,
-    },
-  }),
-  new TralisLayer({
-    isUpdateBboxOnMoveEnd: true,
-    name: 'ch.sbb.puenktlichkeit-fv',
-    visible: false,
-    tenant: 'sbb',
-    getMotsByZoom: (zoom) => (zoom < 4 ? [] : ['rail']),
-    filter: ({ properties }) => {
-      const { type, line } = properties;
-      return (
-        type &&
-        line?.name &&
-        type === 'rail' &&
-        /(IR|IC|EC|RJX|TGV)/.test(line.name)
-      );
-    },
-    properties: {
-      isQueryable: true,
-      popupComponent: 'PunctualityPopup',
-      useTrackerMenu: true,
-    },
-  }),
-];
+// if a permalink parameter is defined (see TralisLayer), we don't display children.
+if (!punctuality.permalinkFilter) {
+  punctuality.children = [
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-gondola',
+      visible: false,
+      properties: {
+        mots: ['gondola'],
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-funicular',
+      visible: false,
+      properties: {
+        mots: ['funicular', 'cablecar'],
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-ferry',
+      visible: false,
+      properties: {
+        mots: ['ferry'],
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-bus',
+      visible: false,
+      properties: {
+        mots: ['bus'],
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-tram',
+      visible: false,
+      properties: {
+        mots: ['tram', 'subway'],
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-nv',
+      visible: false,
+      properties: {
+        filter: ({ properties }) => {
+          const { type, line } = properties;
+          return (
+            type === 'rail' && /^(S|R$|RE|PE|D|IRE|RB|TER)/.test(line?.name)
+          );
+        },
+      },
+    }),
+    new Layer({
+      name: 'ch.sbb.puenktlichkeit-fv',
+      visible: false,
+      properties: {
+        filter: ({ properties }) => {
+          const { type, line } = properties;
+          return type === 'rail' && /(IR|IC|EC|RJX|TGV)/.test(line?.name);
+        },
+      },
+    }),
+  ];
+  punctuality.children.forEach((layer) => {
+    layer.on('change:visible', () => {
```

Comment:
@oterral I'm pretty sure it's a bug in the review app only (I've seen this before), but turning on the child layers does not activate the  parent layers (not only in punctuality), so if punctuality is turned off and then you turn on an mot it doesn't open the web socket connection because TralisLayer visibility is not updated. Double check that it works when using it in WKP. Otherwise LGTM